code/modules/projectiles/guns/projectile/revolver.dm

```diff
@@ -333,6 +333,7 @@
 	sawn_desc = "Omar's coming!"
 	can_holster = FALSE
 	unique_reskin = TRUE
+	var/sawoffable = TRUE
```

Comment:
maybe "sawoff_able"? Something to make it easier to read.

Reply:
can_sawoff?

Reply:
can do